app/Hutch.Relay/Models/Beacon/RequestSummary.cs

```diff
@@ -1,28 +1,44 @@
+using System.Text.Json.Serialization;
 using Hutch.Relay.Config.Beacon;
+using Hutch.Relay.Constants;
 
 namespace Hutch.Relay.Models.Beacon;
 
 public class RequestSummary
 {
-  public string? ApiVersion { get; set; }
+  public string ApiVersion { get; set; } = BeaconApiConstants.ApiVersion;
 
-  public string? Schemas { get; set; }
+  [JsonPropertyName("requestedSchemas")]
+  public List<ReturnedSchema> Schemas { get; set; } = [];
 
   public List<string> Filters { get; set; } = [];
 
-  public string? Parameters { get; set; }
+  [JsonPropertyName("requestParameters")] // TODO: implement for POST
+  public Dictionary<string, string>? Parameters { get; } // TODO: $schema property?
 
-  //public string? IncludeResultSetResponses { get; set; } = ResultsetResponses.Hit; // TODO: We don't support miss today (and i expect never will? but may need to revisit when POST endpoints are added)
+  public string IncludeResultsetResponses { get; set; } = nameof(ResultsetResponses.HIT); // TODO: We don't support miss today (and i expect never will? but may need to revisit when POST endpoints are added)
```

Comment:
Should this be `IncludeResultSetResponses`?

Reply:
No; it was wrong previously.

Per https://b2ri-documentation.readthedocs.io/en/latest/endpoints/individuals.html#tag/GET-Endpoints/operation/getIndividuals


![Screenshot_20250910_094600_Chrome.jpg](https://github.com/user-attachments/assets/50a626e8-ee74-4026-bd9f-cb3565e5908d)